samples/PowerPointModernCommentSample/Program.cs

```diff
@@ -38,64 +38,63 @@ public static void Main(string[] args)
                 }
 
                 // Create a presentation
-                PresentationDocument presentationDocument = PowerPointUtils.CreatePresentation(fileInfo.FullName);
-
-                // create missing PowerPointAuthorsPart if it is null
-                if (presentationDocument?.PresentationPart?.authorsPart is null)
-                {
-                    presentationDocument?.PresentationPart.AddNewPart<PowerPointAuthorsPart>();
-                }
-
-                // Add missing AuthorList if it is null
-                if (presentationDocument?.PresentationPart?.authorsPart?.AuthorList is null)
+                using (PresentationDocument presentationDocument = PowerPointUtils.CreatePresentation(fileInfo.FullName))
```

Comment:
you can replace this with `using PresentationDocument presentationDocument =...;` which doesn't require shifting all the lines

Reply:
don't worry about it here, though. The change on main branch should do it though